packages/flutter_tools/lib/src/native_assets.dart

```diff
@@ -415,3 +437,203 @@ Uri buildUriMultiple(Uri projectUri) {
   final String buildDir = build_info.getBuildDirectory();
   return projectUri.resolve('$buildDir/native_assets/multiple/');
 }
+
+
+/// Dry run the native builds.
+///
+/// This does not build native assets, it only simulates what the final paths
+/// of all assets will be so that this can be embedded in the kernel file.
+Future<Uri?> dryRunNativeAssetsLinuxWindows({
```

Comment:
The "LinuxWindows" naming seems pretty confusing. What is the conceptual thing that makes these the same as each other, but different from iOS/macOS (is it single-arch output files)? That would be a better thing to name it for.

Reply:
I would probably make `*Windows` and `*Linux` versions that are public, and which wrap a private helper named for the conceptual case it handles.